MainRepository/Program.cs

```diff
@@ -1,7 +1,21 @@
-﻿internal class Program
+﻿//Display the individual digits of given number
+while (true)

```

Comment:
Is there a reason why you change for loop to while loop?

Reply:
In most assignments, I used infinite for loop, so I changed to infinite while loop for practice.